src/main/scala/it/stanislas/courses/codergarden/ex1/sol1/Application.scala

```diff
@@ -0,0 +1,16 @@
+package it.stanislas.courses.codergarden.ex1.sol1
```

Comment:
It would be nice to integrate scalafmt in order to have a common formatted everywhere